### ISSUE TITLE
TypeError: Object #<Object> has no method 'addMatchers'

### ISSUE THREAD

**Issue by lynndylanhurley** [zcaudate-me/example.purnam.test#1]
All tests fail with the same error:
![screen shot 2014-03-16 at 7 02 10 pm](https://f.cloud.github.com/assets/468037/2432500/8953ce1e-ad67-11e3-9d85-4bb5fa9b46ba.png)

This is after a fresh clone of this repo. I'm using the following karma packages/versions:

```
"karma": "~0.12.0",
"karma-jasmine": "~0.2.2",
"karma-chrome-launcher": "~0.1.2"
```

**Comment by hoebat** [comment]
you need to include purnam.test in the headers

(ns your.name.space
  (:require [purnam.test]
  ....)

Otherwise this will fail

**Comment by lynndylanhurley** [comment]
I get those errors when I run the tests from this repo. I haven't changed a single line of code. 

Here are the failing tests:
https://gist.github.com/9592665

**Comment by hoebat** [comment]
you may be using an older version of karma that doesn't have jasmine built-in.

**Comment by lynndylanhurley** [comment]
I'm using the latest (v0.12). Which version are you using?

**Comment by hoebat** [comment]
Hmmm... Im using 0.10.\* - an older version. Can you try deleting the karma.js file and doing karma init in the directory to set it up again? It may be that the config file is not compatible

**Comment by lynndylanhurley** [comment]
No luck - I'm still getting the same errors.

This is the file that was generated after I ran `karma init`:
https://gist.github.com/9603651

These are the resulting errors (same as before):
![screen shot 2014-03-17 at 12 08 35 pm](https://f.cloud.github.com/assets/468037/2438865/df92e0f0-adf6-11e3-869d-826758050b1f.png)

**Comment by hoebat** [comment]
That's so strange. I'll install karma later today and see if I can reproduce it.

**Comment by lynndylanhurley** [comment]
Cool, thanks. I'll downgrade to v0.10 and see if that helps.

**Comment by hoebat** [comment]
hmmmm... I've upgraded and its fine I'm still not getting the errors. it may be easier if you email your entire project to me: z <at> caudate.me

**Comment by lynndylanhurley** [comment]
It's literally just this project. Steps to reproduce the problem:

Clone this project and compile using cljsbuild:

```
git clone https://github.com/purnam/example.purnam.test.git
cd example.purnam.test
lein cljsbuild auto
```

In a new window in the same directory:

```
karma start
```

That's it.

Is there any more information that I can send that would help diagnose the issue?

**Comment by hoebat** [comment]
Okay,

I upgraded to the latest version (2.2) of karma-jasmine and I have finally reproduced this error

I was using version 1.4 before and they must have changed it. You can get around it by installing the older one:

npm -g uninstall karma-jasmine
npm -g install karma-jasmine@0.1.4

**Comment by hoebat** [comment]
Yeah I do. I've raised an issue with the jasmine project to figure out what has changed.
